docs/development.md

```
@@ -111,6 +111,49 @@ Do not apply `macos/mask-icon.swift` here. That masks the *app icon* to Apple's
+goes to Apple's unified log instead, so there is somewhere to look afterwards.
+
+Everything the app records is under the subsystem `com.jwp23.throwntom`, in one of five
+categories: `daemon` (requests and the event stream), `service` (launchd, Login Items and other
+requests to macOS), `reminders` (notification permission and banners), `tasks` (the task list and
+its editor) and `stats` (the panel's one fetch). Read the last hour of all of it:
+
+```bash
+/usr/bin/log show --last 1h --predicate 'subsystem == "com.jwp23.throwntom"' --style compact
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)